components/variations/BUILD.gn

```diff
@@ -12,9 +14,12 @@ static_library("variations") {
   sources = [
     "active_field_trials.cc",
     "active_field_trials.h",
-    "android/variations_associated_data_android.cc",
-    "android/variations_seed_bridge.cc",
-    "android/variations_seed_bridge.h",
+
+    # Cobalt only fix! These bring in JNI, which we do not want to bring into Cobalt.
```

Comment:
Same comments as above - consider moving the files into `if` block below.

Reply:
Done